### ISSUE TITLE
Add type to Occurrence

### ISSUE THREAD

**Issue by ocastrup** [OCXStandard/OCX_Schema#98]
Occurrence does not have a type:
```XML
	<xs:element name="Occurrence">
		<xs:annotation>
			<xs:documentation>An occurrence in a DesignView is a reference to a physical part in the model.</xs:documentation>
		</xs:annotation>
		<xs:complexType>
			<xs:choice>
				<xs:element ref="ocx:StructureRef" maxOccurs="unbounded"/>
				<xs:element ref="ocx:StiffenerRef" maxOccurs="unbounded"/>
			</xs:choice>
		</xs:complexType>
	</xs:element>
```

**Comment by ocastrup** [comment]
Add a type definition to Occurrence:
```XML
	<xs:element name="Occurrence">
		<xs:annotation>
			<xs:documentation>An occurrence in a DesignView is a reference to a physical part in the model.</xs:documentation>
		</xs:annotation>
		<xs:complexType>
			<xs:complexContent>
				<xs:extension base="ocx:Occurrence_T"/>
			</xs:complexContent>
		</xs:complexType>
	</xs:element>
	<xs:complexType name="Occurrence_T">
		<xs:annotation>
			<xs:documentation>Type Definition</xs:documentation>
		</xs:annotation>
		<xs:choice>
			<xs:element ref="ocx:StructureRef" maxOccurs="unbounded"/>
			<xs:element ref="ocx:StiffenerRef" maxOccurs="unbounded"/>
		</xs:choice>
	</xs:complexType>
```

**Comment by ocastrup** [comment]
Included in #99
